pom.xml

```diff
@@ -362,6 +362,12 @@
         <artifactId>jclouds-slf4j</artifactId>
         <version>${version.jclouds}</version>
       </dependency>
+      
+      <dependency>
+        <groupId>org.jboss.resteasy</groupId>
+        <artifactId>resteasy-jaxrs</artifactId>
```

Comment:
There is a bug in JClouds and I cannot get needed data from Docker. So I have to get the data from Docker REST. T tried to use "jaxrs-api", but there is no "ClientRequest" class. So I used implementation